app/src/main/graphql/com/example/MediaDetailsById.graphql

```diff
@@ -0,0 +1,21 @@
+query MediaDetailsById($id:Int!) {
+    Media(type: ANIME, id: $id) {
+        id
```

Comment:
Have you worked with GraphQL fragments here? This could be useful to look into: even though the args are different, the fields are exactly the same for the two queries.